docs/04-api-reference/front-commerce-core/make-cache-from-request.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "makeCacheFromRequest"
+sidebar_position: 5
+description: "The makeCacheFromRequest allows to cache response for a request."
```

Comment:
**issue:** this is not correct at all. It's a factory for the caching layer.

Several locations of the documentation below is also incorrect.

**suggestion:** @mGGk-fr do you want to proof-read it, or get detailed feebdack on it (in this case it'll likely be faster for me to do another MR fixing this)

Reply:
Added task FC-1754 for follow up